tests/pc/test_lag_member.py

```diff
@@ -0,0 +1,360 @@
+        lag_port_list.append(port_name)
+        port_list_idx += 1
+
+        if len(lag_port_list) == len(dut_ports[ATTR_PORT_BEHIND_LAG]):
+            break
+
+    duthost.shell("config vlan add {}".format(vlan["id"]))
+    duthost.shell("config interface ip add Vlan{} {}".format(vlan["id"], vlan["ip"]))
+    add_member_to_vlan(duthost, vlan["id"], DUT_LAG_NAME, "-u")
+    remove_member_from_vlan(duthost, 1000, dut_ports[ATTR_PORT_NOT_BEHIND_LAG]["port_name"])
+    remove_ip_from_port(duthost, dut_ports[ATTR_PORT_NOT_BEHIND_LAG]["port_name"])
+    add_member_to_vlan(duthost, vlan["id"], dut_ports[ATTR_PORT_NOT_BEHIND_LAG]["port_name"], "-u")
+
+    lag_port_map = {}
+    lag_port_map[DUT_LAG_NAME] = lag_port_list
+    lag_port_map[ATTR_PORT_NOT_BEHIND_LAG] = dut_ports[ATTR_PORT_NOT_BEHIND_LAG]
+    lag_port_map["vlan"] = vlan
+    return lag_port_map
+
+def setup_ptf_lag(ptfhost, ptf_ports, vlan):
+    """
+    Setup ptf lag
+
+    Args:
+        ptfhost: PTF host object
+        ptf_ports: ports need to configure
+        vlan: information about vlan configuration
+
+    Returns:
+        information about ptf lag
+    """
+    ip_prefix = ".".join(vlan["ip"].split("/")[0].split(".")[0:3])
+    mask_len = vlan["ip"].split("/")[1]
+    lag_ip = "{}.2/{}".format(ip_prefix, mask_len)
+    port_not_behind_lag_ip = "{}.3/{}".format(ip_prefix, mask_len)
+
+    ptfhost.shell("ip link add {} type bond".format(PTF_LAG_NAME))
+    ptfhost.shell("ip link set {} type bond miimon 100 mode 802.3ad".format(PTF_LAG_NAME))
+    ptfhost.shell("ip address add {} dev {}".format(lag_ip, PTF_LAG_NAME))
+    
+    port_list = []
+    for _, port_name in ptf_ports[ATTR_PORT_BEHIND_LAG].items():
+        ptfhost.shell("ip link set {} down".format(port_name))
+        ptfhost.shell("ip link set {} master {}".format(port_name, PTF_LAG_NAME))
+        port_list.append(port_name)
+
+    lag_port_map = {}
+    lag_port_map[PTF_LAG_NAME] = {
+        "port_list": port_list,
+        "ip": lag_ip
+    }
+    lag_port_map[ATTR_PORT_NOT_BEHIND_LAG] = ptf_ports[ATTR_PORT_NOT_BEHIND_LAG]
+    lag_port_map[ATTR_PORT_NOT_BEHIND_LAG]["ip"] = port_not_behind_lag_ip
+    
+    ptfhost.shell("ip link set dev {} up".format(PTF_LAG_NAME))
+    ptfhost.shell("ifconfig {} mtu 9100 up".format(PTF_LAG_NAME))
+    ptfhost.shell("ip addr add {} dev {}".format(port_not_behind_lag_ip, ptf_ports[ATTR_PORT_NOT_BEHIND_LAG]["port_name"]))
+    restart_ptf_nn_agent(ptfhost)
+    time.sleep(10)
+
+    return lag_port_map
+
+def setup_dut_ptf(ptfhost, duthost, tbinfo):
+    """
+    Setup dut and ptf
+
+    Args:
+        ptfhost: PTF host object
+        duthost: DUT host object
+        tbinfo: fixture provides information about testbed
+    
+    Returns:
+        information about lag of ptf and dut
+    """
+    cfg_facts = duthost.config_facts(host = duthost.hostname, source="running")["ansible_facts"]
```

Comment:
Add some comments here? Why you want to grab pc members from config?

Reply:
Done

Reply:
Done